contracts/modules/licensing/LicensingModule.sol

```diff
@@ -158,20 +163,26 @@ contract LicensingModule is
         }
 
         _verifyIpNotDisputed(licensorIpId);
-
-        Licensing.MintingLicenseConfig memory mlc = LICENSE_REGISTRY.verifyMintLicenseToken(
+        Licensing.LicensingConfig memory lsc = LICENSE_REGISTRY.verifyMintLicenseToken(
             licensorIpId,
             licenseTemplate,
             licenseTermsId,
             _hasPermission(licensorIpId)
         );
-        if (mlc.receiverCheckModule != address(0)) {
-            if (!IHookModule(mlc.receiverCheckModule).verify(receiver, mlc.receiverCheckData)) {
-                revert Errors.LicensingModule__ReceiverCheckFailed(receiver);
-            }
+        uint256 mintingFeeByHook = 0;
```

Comment:
same

Reply:
yes. as explained in the above comments.